delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/acls/Acls.scala

```diff
@@ -0,0 +1,109 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.model.acls
+
+import ch.epfl.bluebrain.nexus.delta.sdk.AclResource
+import ch.epfl.bluebrain.nexus.delta.sdk.model.Caller
+import ch.epfl.bluebrain.nexus.delta.sdk.model.acls.AclRejection.RevisionNotFound
+import monix.bio.{IO, Task}
+
+/**
+  * Operations pertaining to managing Access Control Lists.
+  */
+trait Acls {
+
+  /**
+    * Fetches the ACL resource for a ''target'' on the current revision.
+    *
+    * @param target the target location for the ACL
+    */
+  def fetch(target: Target): Task[Option[AclResource]]
+
+  /**
+    * Fetches the ACL resource for a ''target'' on the passed revision.
+    *
+    * @param target the target location for the ACL
+    * @param rev    the revision to fetch
+    */
+  def fetchAt(target: Target, rev: Long): IO[RevisionNotFound, Option[AclResource]]
+
+  /**
+    * Fetches the ACL resource for a ''target'' on the current revision.
+    *
+    * @param target the target location for the ACL
+    * @param self   flag to decide whether or not ACL of other identities than the provided ones should be included in the response.
+    *               This is constrained by the current caller having ''acls/read'' permissions on the provided ''target'' or it's parents
+    */
+  final def fetch(target: Target, self: Boolean)(implicit caller: Caller): Task[Option[AclResource]] =
```

Comment:
I would make this function explicit: `def fetchSelf` since the routes should discriminate between the intent. This way we can explicitly ask for a caller argument and we can drop the self argument.

Reply:
Sure. Even though that's gonna be more code on the routes side, since `self` is a query parameter, we'll have to do:
```scala
parameters("self" ? false) { self =>
   if(self) fetchSelf(target)
   else fetch(target)
}
```


```